src/web_app_skeleton/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3.8'
+services:
+  lucky:
+    build:
+      context: .
+      dockerfile: docker/development.dockerfile
+    environment:
+      DATABASE_URL: postgres://postgres@postgres:5432/postgres
+    volumes:
+      - type: bind
+        source: .
+        target: /app
+      - type: volume
+        source: node_modules
+        target: /app/node_modules
+      - type: volume
+        source: shards_lib
+        target: /app/lib
+    depends_on:
+      - postgres
+
+    entrypoint: ["docker/dev_entrypoint.sh"]
```

Comment:
I opted to embed the entrypoint in a script rather than simply inline the startup logic to keep this file terse. In the long run I think the entrypoint script will likely grow to be more helpful and I don't think it'll live inline here reasonably very well.

Reply:
Usually these are in the Dockerfile, so the same image could be run standalone without a rebuild

Reply:
I started with it there, actually. I removed it because this dockerfile is intended to be used with the compose, and the file doesn't exist without the compose mounted volume. If the entrypoint were coded into the docker file, the file would also have to be copied into the container as well.